src/main/java/org/sagebionetworks/bridge/sdk/LowercaseEnumModule.java

```diff
@@ -0,0 +1,34 @@
+package org.sagebionetworks.bridge.sdk;
+
+import com.fasterxml.jackson.core.Version;
+import com.fasterxml.jackson.databind.BeanDescription;
+import com.fasterxml.jackson.databind.DeserializationConfig;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.deser.Deserializers;
+import com.fasterxml.jackson.databind.deser.Deserializers.Base;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+@SuppressWarnings("serial")
+class LowercaseEnumModule extends SimpleModule {
```

Comment:
The UserSessionTest only tests deserialization, and I see no other tests for serialization or deserialization. We should add some unit tests to make sure this thing works.